### PR TITLE
tags for open content. no special names

### PR DESCRIPTION
**Pull request by oganm**
This adds source tags for SRD content as discussed in #103. Currently It does not mark spells with special names (eg. Melf) as I didn't know what to do with them. They can be added on request.

These files are automatically generated using data from [open5e](https://github.com/eepMoody/open5e). Code for generation can be found [here](https://gist.github.com/oganm/f8fc5fea789aa2a6a436dcbe51198006). Had to edit one file (antipathy/sympathy) by hand due to a mismatch in their naming.

You see some changes in the ends of a few files. Those files did not have newlines at the end originally. A newline is added them during the process. ~~Since most files already seem to have this I didn't think this is an issue~~. Style guide says files should have a newline at the end so I didn't try to fix that

### CODE REVIEW

**Comment by Traneptora** [comment]
Include page numbers in the System Reference Document. The notation is `SRD.56` for SRD, page 56.

**Comment by oganm** [comment]
What do you want to do with Melf et al?

**Comment by Traneptora** [comment]
I'm not familiar with the extra sources for Melf et. al. (I didn't know those were a thing until recently :o). Either way, to add them as a source, I'd prefer it to be a separate pull request, since smaller, more modular commits are generally better on principle.

**Comment by oganm** [comment]
Oh no I wasn't clear enough. I meant what should be done if a spell is technically SRD but it's name isn't. For instance SRD name for "Melf's acid arrow" is, "acid arrow". This is true for all spells with people's names in it. Currently have not marked them as SRD but I can do that.

**Comment by Traneptora** [comment]
I can think of three solutions than includes all the data:
1. Mark them as SRD spells using the PHB names
2. Mark them as SRD spells and implement a "multiple names" system
3. Enter them as separate spells.

I think the first solution is definitely the best and as the "maintainer" of this current fork of The Grimoire I'm going to recommend just using the PHB (et. al.) names.

**Comment by oganm** [comment]
Added page nos.

**Comment by oganm** [comment]
I believe this can be pulled now

File: _posts/2014-08-24-acid-splash.markdown [hunk 1]
```diff
@@ -18,4 +18,4 @@ tags: [sorcerer, wizard, cantrip, conjuration]
 
 You hurl a bubble of acid. Choose one creature within range, or choose two creatures within range that are within 5 feet of each other. A target must succeed on a Dexterity saving throw or take 1d6 acid damage.
 
-This spell’s damage increases by 1d6 when you reach 5th level (2d6), 11th level (3d6), and 17th level (4d6).
\ No newline at end of file
+This spell’s damage increases by 1d6 when you reach 5th level (2d6), 11th level (3d6), and 17th level (4d6).
```
Comment (Traneptora) [inline _posts/2014-08-24-acid-splash.markdown:21]: What changed  here?

File: _posts/2014-08-24-charm-person.markdown [hunk 2]
```diff
@@ -18,4 +18,4 @@ tags: [bard, cleric (trickery), druid, sorcerer, warlock, wizard, level1, enchan
 
 You attempt to charm a humanoid you can see within range. It must make a Wisdom saving throw, and does so with advantage if you or your companions are fighting it. If it fails the saving throw, it is charmed by you until the spell ends or until you or your companions do anything harmful to it. The charmed creature regards you as a friendly acquaintance. When the spell ends, the creature knows it was charmed by you.
 
-**At Higher Levels.** When you cast this spell using a spell slot of 2nd level or higher, you can target one additional creature for each slot level above 1st. The creatures must be within 30 feet of each other when you target them.
\ No newline at end of file
+**At Higher Levels.** When you cast this spell using a spell slot of 2nd level or higher, you can target one additional creature for each slot level above 1st. The creatures must be within 30 feet of each other when you target them.
```
Comment (Traneptora) [inline _posts/2014-08-24-charm-person.markdown:21]: What changed here?

File: _posts/2014-08-24-color-spray.markdown [hunk 3]
```diff
@@ -20,4 +20,4 @@ A dazzling array of flashing, colored light springs from your hand. Roll 6d10; t
 
 Starting with the creature that has the lowest current hit points, each creature affected by this spell is blinded until the spell ends. Subtract each creature’s hit points from the total before moving on to the creature with the next lowest hit points. A creature’s hit points must be equal to or less than the remaining total for that creature to be affected.
 
-**At Higher Levels.** When you cast this spell using a spell slot of 2nd level or higher, roll an additional 2d10 for each slot level above 1st.
\ No newline at end of file
+**At Higher Levels.** When you cast this spell using a spell slot of 2nd level or higher, roll an additional 2d10 for each slot level above 1st.
```
Comment (Traneptora) [inline _posts/2014-08-24-color-spray.markdown:23]: What changed here?

File: _posts/2014-08-24-detect-evil-and-good.markdown [hunk 4]
```diff
@@ -18,4 +18,4 @@ tags: [cleric, paladin, level1, divination]
 
 For the duration, you know if there is an aberration, celestial, elemental, fey, fiend, or undead within 30 feet of you, as well as where the creature is located. Similarly, you know if there is a place or object within 30 feet of you that has been magically consecrated or desecrated.
 
-The spell can penetrate most barriers, but it is blocked by 1 foot of stone, 1 inch of common metal, a thin sheet of lead, or 3 feet of wood or dirt.
\ No newline at end of file
+The spell can penetrate most barriers, but it is blocked by 1 foot of stone, 1 inch of common metal, a thin sheet of lead, or 3 feet of wood or dirt.
```
Comment (Traneptora) [inline _posts/2014-08-24-detect-evil-and-good.markdown:21]: What changed here?

File: _posts/2014-08-24-disguise-self.markdown [hunk 5]
```diff
@@ -20,4 +20,4 @@ You make yourself -- including your clothing, armor, weapons, and other belongin
 
 The changes wrought by this spell fail to hold up to physical inspection. For example, if you use this spell to add a hat to your outfit, objects pass through the hat, and anyone who touches it would feel nothing or would feel your head and hair. If you use this spell to appear thinner than you are, the hand of someone who reaches out to touch you would bump into you while it was seemingly still in midair.
 
-To discern that you are disguised, a creature can use its action to inspect your appearance and must succeed on an Intelligence (Investigation) check against your spell save DC.
\ No newline at end of file
+To discern that you are disguised, a creature can use its action to inspect your appearance and must succeed on an Intelligence (Investigation) check against your spell save DC.
```
Comment (Traneptora) [inline _posts/2014-08-24-disguise-self.markdown:23]: What changed here?

File: _posts/2014-08-24-druidcraft.markdown [hunk 6]
```diff
@@ -21,4 +21,4 @@ Whispering to the spirits of nature, you create one of the following effects wit
 * You create a tiny, harmless sensory effect that predicts what the weather will be at your location for the next 24 hours. The effect might manifest as a golden orb for clear skies, a cloud for rain, falling snowflakes for snow, and so on. This effect persists for 1 round.
 * You instantly make a flower bloom, a seed pod open, or a leaf bud bloom.
 * You create an instantaneous, harmless sensory effect, such as falling leaves, a puff of wind, the sound of a small animal, or the faint order of skunk. The effect must fit in a 5-foot cube.
-* You instantly light or snuff out a candle, a torch, or a small campfire.
\ No newline at end of file
+* You instantly light or snuff out a candle, a torch, or a small campfire.
```
Comment (Traneptora) [inline _posts/2014-08-24-druidcraft.markdown:24]: What changed here?

File: _posts/2014-08-24-eldritch-blast.markdown [hunk 7]
```diff
@@ -18,4 +18,4 @@ tags: [warlock, cantrip, evocation]
 
 A beam of crackling energy streaks toward a creature within range. Make a ranged spell attack against the target. On a hit, the target takes 1d10 force damage.
 
-The spell creates more than one beam when you reach higher levels: two beams at 5th level, three beams at 11th level, and four beams at 17th level. You can direct the beams at the same target or at different ones. Make a separate attack roll for each beam.
\ No newline at end of file
+The spell creates more than one beam when you reach higher levels: two beams at 5th level, three beams at 11th level, and four beams at 17th level. You can direct the beams at the same target or at different ones. Make a separate attack roll for each beam.
```
Comment (Traneptora) [inline _posts/2014-08-24-eldritch-blast.markdown:21]: What changed here?

File: _posts/2014-08-24-entangle.markdown [hunk 8]
```diff
@@ -20,4 +20,4 @@ Grasping weeds and vines sprout from the ground in a 20-foot square starting for
 
 A creature in the area when you cast the spell must succeed on a Strength saving throw or be restrained by the entangling plants until the spell ends. A creature restrained by the plants can use its action to make a Strength check against your spell save DC. On a success, it frees itself.
 
-When the spell ends, the conjured plants wilt away.
\ No newline at end of file
+When the spell ends, the conjured plants wilt away.
```
Comment (Traneptora) [inline _posts/2014-08-24-entangle.markdown:23]: What changed here?

File: _posts/2014-08-24-expeditious-retreat.markdown [hunk 9]
```diff
@@ -16,4 +16,4 @@ tags: [sorcerer, warlock, wizard, level1, transmutation]
 
 **Duration**: Concentration, up to 10 minutes
 
-This spell allows you to move at an incredible pace. When you cast this spell, and then as a bonus action on each of your turns until the spell ends, you can take the Dash action.
\ No newline at end of file
+This spell allows you to move at an incredible pace. When you cast this spell, and then as a bonus action on each of your turns until the spell ends, you can take the Dash action.
```
Comment (Traneptora) [inline _posts/2014-08-24-expeditious-retreat.markdown:19]: What changed here?

File: _posts/2014-08-24-feather-fall.markdown [hunk 10]
```diff
@@ -16,4 +16,4 @@ tags: [bard, sorcerer, wizard, level1, transmutation]
 
 **Duration**: 1 minute
 
-Choose up to five falling creatures within range. A falling creature’s rate of descent slows to 60 feet per round until the spell ends. If the creature lands before the spell ends, it takes no falling damage and can land on its feet, and the spell ends for that creature.
\ No newline at end of file
+Choose up to five falling creatures within range. A falling creature’s rate of descent slows to 60 feet per round until the spell ends. If the creature lands before the spell ends, it takes no falling damage and can land on its feet, and the spell ends for that creature.
```
Comment (Traneptora) [inline _posts/2014-08-24-feather-fall.markdown:19]: What changed here?

File: _posts/2014-08-24-goodberry.markdown [hunk 11]
```diff
@@ -18,4 +18,4 @@ tags: [druid, ranger, level1, transmutation]
 
 Up to ten berries appear in your hand and are infused with magic for the duration. A creature can use its action to eat one berry. Eating a berry restores 1 hit point, and the berry provides enough nourishment to sustain a creature for a day.
 
-The berries lose their potency if they have not been consumed within 24 hours of the casting of this spell.
\ No newline at end of file
+The berries lose their potency if they have not been consumed within 24 hours of the casting of this spell.
```
Comment (Traneptora) [inline _posts/2014-08-24-goodberry.markdown:21]: What changed here?

File: _posts/2014-08-24-grease.markdown [hunk 12]
```diff
@@ -18,4 +18,4 @@ tags: [wizard, level1, conjuration]
 
 Slick grease covers the ground in a 10-foot square centered on a point within range and turns it into difficult terrain for the duration.
 
-When the grease appears, each creature standing in its area must succeed on a Dexterity saving throw or fall prone. A creature that enters the area or ends its turn there must also succeed on a Dexterity saving throw or fall prone.
\ No newline at end of file
+When the grease appears, each creature standing in its area must succeed on a Dexterity saving throw or fall prone. A creature that enters the area or ends its turn there must also succeed on a Dexterity saving throw or fall prone.
```
Comment (Traneptora) [inline _posts/2014-08-24-grease.markdown:21]: What changed here?

**Comment by Traneptora** [comment]
As I was going through it occurred to me there's a few lines that changed that you didn't appear to do anything with. I believe it's an issue with line endings. Doublecheck those (`dos2unix` is useful). I stopped commenting when I realized that might be it, so I didn't point all of them out.

**Comment by oganm** [comment]
A side effect of the script I used was that it added a newline at the end of every file it touched if it didn't already have one. Since your guidelines suggested there should have been a newline to begin with, I didn't "fix" that quirk (see first post).

**Comment by Traneptora** [comment]
My bad, sorry. That is fine.

**Pull request merged**